charts/perses/values.yaml

```diff
@@ -226,3 +226,12 @@ datasources:
   #       kind: PrometheusDatasource
   #       spec:
   #         directUrl: https://prometheus.demo.do.prometheus.io
+
+
+testFramework:
+  enabled: true
+  image:
+    registry: ghcr.io
+    repository: cloudoperators/greenhouse-extensions-integration-test
+    tag: main
+  imagePullPolicy: IfNotPresent
```

Comment:
(nit) can you add a breakline?

Reply:
fixed in 3cc01c3022c398e246f91811e0ea4d030666e83d